### DISCUSSION TITLE
Change the size of pinned tab Favicons?

### DISCUSSION THREAD

**Discussion by Bout2GitIt** [piroor/treestyletab#3359] [Q&A] [answered]
My goal: Increase the size of pinned tabs _and the favicon_.

I was able to change the size of my pinned tabs using the faviconizedTabScale option in the All Configs menu.
![Screen Shot 2023-07-05 at 11 21 48 AM](https://github.com/piroor/treestyletab/assets/45543439/13931833-b732-40c2-8ed3-8f91153bd73c)

But the size of the favicons hasn't scaled to match the size of the pinned tab; the favicons are still small. 
![Screen Shot 2023-07-07 at 10 08 19 AM](https://github.com/piroor/treestyletab/assets/45543439/fa65452e-5b8a-4ecc-aefa-cc6863cc06da)

Is there a way to increase the size of the favicons on the pinned tabs?

**Comment by irvinm** [comment, marked as answer]
I think this would work.  Adjust as desired.

```
tab-item.pinned tab-favicon {
	transform: scale(2.0) !important;
}
```

**Reply by Bout2GitIt** [reply]
This worked great, thank you!
